public/app/features/dashboard/components/HelpWizard/SupportSnapshotService.ts

```diff
@@ -82,7 +82,7 @@ export class SupportSnapshotService extends StateManagerBase<SupportSnapshotStat
 
     if (!panel.isAngularPlugin()) {
       try {
-        const oldModel = new DashboardModel(snapshot);
+        const oldModel = new DashboardModel(snapshot, { isEmbedded: true });
         const dash = createDashboardSceneFromDashboardModel(oldModel);
```

Comment:
I've noticed one panel (Data from panel above)  is not showing data in the snapshot preview, not sure if it's related to the changes in this PR:

<img width="1707" alt="image" src="https://github.com/grafana/grafana/assets/2376619/8e7a7480-f399-4776-b970-5c6843cbccb4">

Reply:
@dprokop that looks to be unrelated, a bug in SceneQueryRunner.getResultStream , testing a fix now